sdk-setup/src/sdk-manage

```diff
@@ -1565,13 +1565,23 @@ target_snapshot_needs_reset() {
         return 1
     fi
 
-    # TODO how to determine recent package removal?
     local most_recent_install_time=
     most_recent_install_time=$(set -o pipefail; rpm --root="$MER_TARGETS/$original" -qa \
-        --queryformat '%{installtime}\n' |sort -n |tail -n1) || return
-    local snapshot_time_s=$(date --date "$snapshot_time" +%s) || return
+        --queryformat '%{installtime}\n' | sort -n | tail -n1) || return
```

Comment:
That |is |the |style |used |here. Please keep it and follow in the new code :)

Reply:
Updated to keep the style